### PR TITLE
Apply JS-best practices for To-do List project

### PR DESCRIPTION
**Pull request by kalbek**
On this version of the pull request, best practices for Javascript have been applied.

### CODE REVIEW

File: src/modules/Todo.js [hunk 1]
```diff
@@ -0,0 +1,237 @@
+export default class Todo {
+  // todo list arrat to store all todos
+  static todoList = [
+    {
+      description: 'first todo . . . ',
+      completed: false,
+      index: 0,
+    },
```
Comment (hasnatali1947) [inline src/modules/Todo.js:8]: remove this unnecessary object. because todo needs to be empty initially

File: src/modules/Todo.js [hunk 2]
```diff
@@ -0,0 +1,237 @@
...
+    Todo.updateUI(todoList);
+    // update localstorage for todos
+    Todo.updateLocalstorage();
+  };
+
+  // update todo as a whole (especially) from localstorage
+  static updateTodo = (currentTodo) => {
+    Todo.todoList = currentTodo;
+    Todo.updateUI(document.querySelector('.list-container'));
+  };
```
Comment (hasnatali1947) [inline src/modules/Todo.js:37]: you are not going to need these updated todo functions because you have not used them anywhere.

**Review by hasnatali1947 — CHANGES_REQUESTED** [review]

File: src/modules/Todo.js (hunk 2 above)
Comment (kalbek) [reply src/modules/Todo.js:37]: That's right, 👍 Thank you for picking that out. I almost forgot it: 🤯

**Review by hasnatali1947 — APPROVED** [review]
great job 😄 
it's time to merge it 👍

**Pull request merged**